.travis.yml

```diff
@@ -11,17 +11,24 @@ cache:
   directories:
     - $HOME/.composer/cache
 
+stages:
+  - composer validate
+  - test
+  - test with coverage
+  - check coding style
```

Comment:
Would it be nice to move some of the "faster" steps to run before the slower ones (run test, test with coverage and integration tests last)?

If the build is going to fail, would be nice if it fails fast.

Reply:
changed